### ISSUE TITLE
...Is the project dead?

### ISSUE THREAD

**Issue by Nariod14** [MASGAU/MASGAU#115]
I haven't been able to use my c for a while due to hard drive problems, and now that I've just got it fixed (somewhat) I don't trust my gamesaves not to corrupt. I've used this amazing program since last I had my PC working but I've noticed the site is down and my previous issue is the latest one (still from last year) so if I may, is the project dead? I can only find other versions on CNet and other sites but they only go up to the 2013 updates. I can use those, as it is still an amazing program but I just need some closure on what's going on if I could.

**Comment by AndrewSav** [comment]
@Nariod14  There has not been a push for 4 years, what do you think? Somebody started [masgau2](https://github.com/MASGAU/2.0) but it was never completed either.

**Comment by AdmiringWorm** [comment]
as @AndrewSav says, I believe this project is pretty much dead.

> Somebody started masgau2 but it was never completed either.

Not just somebody, it was the author of this software that started that one.

EDIT:
Although it says member on me here, I'm was pretty much a contributor and nothing else.

**Comment by sanmadjack** [comment]
I am the original author.

I live, but yes I don't work on this anymore. I've been keeping the site up because, as you say, the program does still work, even if it does have a few issues still. I plan on revisiting it to fix some bugs someday, but I have projects I am working on now that are much more urgent to me. If someone felt motivated to take over development of the app I would be more than happy to coach them in doing so, and to continue hosting the site on my server.

@AdmiringWorm I somehow had not noticed the pull request, I'll try to take a look at this weekend. I can't say I'll make a new release build for it, but I can at least merge in the code. 

As for the site being down, the digitalocean droplet it's hosted on locked up for some reason. I just restarted it and the site is available again. @Nariod14 Thank you for reporting the site being down.

TLDR: Site is back up, my time for working on this is down. Unless someone feels like taking over, then the project is effectively dead until an indeterminate point far in the future.

**Comment by Nariod14** [comment]
@sanmadjack alright it's nice to have confirmation on that. I can't really complain as the current product is still extremely useful and functional. Thanks for bringing the site back up though I'll get the latest version from there.  I would love to be a part of this but I've barely any experience with any type of code aside from Javascript (sad I know). Best of luck to your current and future endeavors mate.

**Comment by sanmadjack** [comment]
@Nariod14 Same to you. And thanks for the kind words.

**Comment by sigmap** [comment]
Site is dead again?

**Comment by sanmadjack** [comment]
The VM is having issues, I'm working on it now.

**Comment by sanmadjack** [comment]
The root drive's uuid changed for some reason. Corrected it, site is back up.

**Comment by d0x360** [comment]
I've moved on to game save manager 3 which works much better and the guy is working on gsm4 right now which so far is absurdly fast even with thousands of saves

**Comment by AndrewSav** [comment]
I don't think that  game save manager is open source.

**Comment by d0x360** [comment]
I'm working on that actually and I believe I'm very close to getting Matt
the developer to open source it.  Right now he's very busy trying to finish
v4 and once that's essentially at a beta level it should go up here.

**Comment by AndrewSav** [comment]
@d0x360, how is this version 4 coming together?

**Comment by d0x360** [comment]
> @d0x360, how is this version 4 coming together?

Slow but amazing.  The app is insanely fast.  For example in v3 because Ive played so many games it would take me like 45 min from hitting backup and then waiting for it to be done.

In v4 it's more like 15 min and if I turned off max compression it would probably be half that.

There are test builds on the discord channel if you wanna check it out.

**Comment by AndrewSav** [comment]
@d0x360 thank you I'll check with you in another year ;)
